Project.toml

```diff
@@ -12,8 +13,10 @@ FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
 FilePathsBase = "48062228-2e41-5def-b9a4-89aafe57970f"
 Inflate = "d25df0c9-e2be-5dd7-82c8-3ad0b3e990b9"
 Interpolations = "a98d9a8b-a2ab-59e6-89dd-64a1c18fca59"
+JLD2 = "033835bb-8acc-5ee8-8aae-3f567f8a3819"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+LinearMaps = "7a12625a-238d-50fd-b39a-03d52299707e"
```

Comment:
Every additional package will affect load time. It seems like JLD2 will bring a lot of value, but maybe we can work around LinearMaps?

Reply:
LinearMaps seems relatively lightweight: 
https://juliahub.com/ui/Packages/LinearMaps/bdfwy/3.10.2?page=1

If you think it will make the interface more convenient then I'm on OK with including this package.

Reply:
Yep, it's lightweight and in some sense the "correct" way to do certain kinds of linear algebra in Julia. We're bound to need it at some point